### DISCUSSION TITLE
Filing an extended-license project

### DISCUSSION THREAD

**Discussion by akauppi** [google/devlibrary#221] [General]
I really like the way Dev Library uses the GitHub APIs for enquiring a project's license.

However, sometimes one has licenses that don't quite fit the model. A repo I'm going to file is MIT licensed, except its graphic art which I expect people to replace with their own.

How would you file this?

---

I already use Git submodules in the repo, so am thinking of making a submodule just for the artwork. That submodule has its own LICENSE but this allows me to declare the repo itself as a generic MIT text, and boom - GitHub and Dev Library should be pleased.

**Comment by samtstern** [comment]
@akauppi right now we just believe whatever GitHub says, we don't have any plans to do something more advanced than that. However you're not the first person to ask so I've filed an issue for us to eventually add the ability to have a human review the license and override our API check: https://github.com/google/devlibrary/issues/222

**Reply by akauppi** [reply]
Hi @samtstern

Thanks for the heads up. However, I don't have a problem implementing the submodule thing described above. So.. if it's for others, sure. I'll be able to comply with the existing rules. Just… the code still needs some care before that.
